docs/topics_5/web-openshift-troubleshoot.adoc

```diff
@@ -72,9 +72,9 @@ $ oc logs <POD_NAME> > ./mta-openshift-<POD_NAME>.log
 
 == Common issues
 
-.Executor Pod throws `NoRouteToHostException`
+.The `executor` pod throws `NoRouteToHostException`
```

Comment:
"Throws" is a bit too colloquial for technical writing. Suggest you change title to "`NoRouteToHostException` error"

"When accessing..." is not necessary. Suggested rewrite: "The following error message might appear in the `mta-web-console-executor` pod log.

Please check the names of all these pods with Marco Rizzo or Jonathan Vila. I know that this is the correct name for this pod but I can't confirm all of them. 

Also, you need to do this asap because many people will take off time before official shutdown.